typings/found.d.ts

```diff
@@ -11,3 +11,11 @@ declare module "found" {
 
   const ResolverUtils: ResolverUtils
 }
+
+declare module "farce" {
+  type HistoryProtocol = "browser" | "hash" | "memory"
+
+  interface HistoryOptions {
+    useBeforeUnload?: boolean
+  }
+}
```

Comment:
Since these two modules go together I kept them in the same file, but can break out into `farce.d.ts` if this seems weird.

Reply:
Seems fine to me 👍

Just try to PR upstream ASAP!